### ISSUE TITLE
False negative when unquoted name/strings are passed in objects

### ISSUE THREAD

**Issue by rbairwell** [Seldaek/jsonlint#60]
With the test json:
```json
{ 	nonceValue:"90a9225e1ac785aea7c9e80f232d21a6" }
```
(yep, just that)

PHP's own json_decode returns `Syntax error`, it looks like Github's markdown display system also detects the issue, and  https://jsonlint.com/ returns
```
Error: Parse error on line 1:
{	nonceValue: "90a9225
--^
Expecting 'STRING', '}', got 'undefined'
```

However, the linter (release 1.7) fails to detect an issue.

**Comment by Seldaek** [comment]
I get this:

```
$ bin/jsonlint test.json
test.json: Parse error on line 1:
{       nonceValue:"90a9225e
--^
Expected one of: 'STRING', '}'
```

.. which seems correct. Can you check with 1.7.1 and if you still get the problem let me know how to reproduce?

**Comment by rbairwell** [comment]
Thanks. I think it might have been a misunderstanding on my end. PHPstorm said the return definition of ->lint was ` @return null|ParsingException` which I misread as it'll return null or *throw* ParsingException.

My mistake entirely it seems. Sorry about that!

**Comment by Seldaek** [comment]
I see :) No worries
